### PR TITLE
Update Images

### PR DESCRIPTION
**Pull request by Kartik494**
**What type of PR is this?**

> /kind bug

**What this PR does / why we need it**:

In Accordance to #339 update the images for specific version v2.1.0

**Does this PR introduce a user-facing change?**:

```
NONE
```

### CODE REVIEW

File: deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
+          image: quay.io/k8scsi/csi-snapshotter:v2.1.0
```
Comment (ggriffiths) [inline deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml:87]: Can you update these to the latest patch versions? v2.1.1 for both

Comment (Kartik494) [reply deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml]: @ggriffiths I have addressed the changes Please take a look

File: deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml [hunk 2]
```diff
@@ -84,7 +84,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
+          image: quay.io/k8scsi/csi-snapshotter:v2.1.1
```
Comment (xing-yang) [inline deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml:87]: k8s.gcr.io/sig-storage/csi-snapshotter:v2.1.1

File: deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccount: snapshot-controller
       containers:
         - name: snapshot-controller
-          image: quay.io/k8scsi/snapshot-controller:v2.0.1
+          image: quay.io/k8scsi/snapshot-controller:v2.1.1
```
Comment (xing-yang) [inline deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml:22]: k8s.gcr.io/sig-storage/snapshot-controller:v2.1.1

**Comment by xing-yang** [comment]
@ggriffiths @Kartik494 For released images, we should use those from the new repos.

**Comment by xing-yang** [comment]
Please give a try and make sure you can download them.

**Comment by ggriffiths** [comment]
Yeah, v2.1.1 works on gcr.io/sig-storage:
```
➜  ~ docker pull k8s.gcr.io/sig-storage/csi-snapshotter:v2.1.1


v2.1.1: Pulling from sig-storage/csi-snapshotter
Digest: sha256:de94e695fc27ee421da8b9932c3d42699fe7b848bc34717f67f4350946bc46c9
Status: Image is up to date for k8s.gcr.io/sig-storage/csi-snapshotter:v2.1.1
```

Canary doesn't exist yet as you mentioned.
```
➜  ~ docker pull k8s.gcr.io/sig-storage/csi-snapshotter:canary


Error response from daemon: manifest for k8s.gcr.io/sig-storage/csi-snapshotter:canary not found
```

**Comment by Kartik494** [comment]
@xing-yang @ggriffiths i have update the images to k8s.gcr.io/sig-storage/csi-snapshotter:v2.1.1 and k8s.gcr.io/sig-storage/snapshot-controller:v2.1.1 in respective yaml files

**Comment by yuxiangqian** [comment]
/approve cancel
please take a look at the failing test

**Comment by Kartik494** [comment]
Hi @xing-yang @yuxiangqian This is failing for installation of go1.15  in pipe i am not able to configure it yet, can you please give some inputs on why this is failing in installation of go1.15.
Thanks

**Comment by Kartik494** [comment]
Hi @ggriffiths i have faced ("ERROR: installation of Go 1.15.0 failed") above also . could you please tell is this issue related to this Job only or is there any changes to be done in commit?
Thanks

**Comment by ggriffiths** [comment]
Hi @Kartik494, I don't see anything in your commit that would cause a go build issue. Let me open a test PR against release-2.1 and see if there's a problem there.

**Comment by ggriffiths** [comment]
@Kartik494, seems like there is an issue with release-2.1.
https://github.com/kubernetes-csi/external-snapshotter/pull/384

I've opened https://github.com/kubernetes-csi/external-snapshotter/issues/385 to track this and am looking into it now.

**Comment by Kartik494** [comment]
@ggriffiths Yes i got this error for 2.1 release only, Thanks for looking into this

**Pull request merged**